src/components/Calculator.js

```diff
@@ -29,11 +29,8 @@ function Calculator() {
     return btns;
   };
 
-  const keyboardHandler = () => {
-  };
-
   return (
-    <div className="calculator" role="button" tabIndex="0" onClick={Handler} onKeyDown={keyboardHandler}>
+    <div className="calculator" role="button" tabIndex="0" onClick={Handler} onKeyDown={Handler}>
       <div className="result">
         {total}
         {operation}
```

Comment:
- [ ] I suggest that you refactor using the arrow functions avoid using `function` keyword to better practice ES6 as a modern tool to use and it makes your codes more readable you can simply do:
```
const Calculator = () => (
    // reference as a js object
      <div>
       // Calculator app design goes here
      </div>
);
```